docs/source/tools/assistant.rst

```diff
@@ -1,312 +1,155 @@
-    -v,--version             Show version and exit
-
-  Available commands:
-    status                   Show SDK environment overview
-    docs                     Show SDK documentation
-    new                      Create a new project from template
-    add                      Add a template to the current project
-    project                  Manage DAML SDK projects
-    template                 Manage DAML SDK templates
-    upgrade                  Upgrade to latest SDK version
-    list                     List installed SDK versions
-    use                      Set the default SDK version, downloading it if
-                             necessary
-    uninstall                Remove DAML SDK versions or the complete DAML SDK
-    start                    Start a given service
-    restart                  Restart a given service
-    stop                     Stop a given service
-    feedback                 Send us feedback!
-    studio                   Start DAML Studio in the current project
-    navigator                Start Navigator (also runs Sandbox if needed)
-    sandbox                  Start Sandbox process in current project
-    compile                  Compile a DAML project into a DAR package
-    path                     Show the filesystem path of an SDK component
-    run                      Run the main executable of a package
-    setup                    Set up SDK environment (e.g. on install or upgrade)
-    subscribe                Subscribe for a namespace (of the template
-                             repository).
-    unsubscribe              Unsubscribe from a namespace (of the template
-                             repository).
-    config-help              Show config-file help
-    config                   Query and manage configuration
-    changelog                Show the changelog of an SDK version
-    update-info              Show SDK Assistant update channel information
-
-To get help for a particular command, use this command::
-
-  da <command> --help
-
-**Example**::
-
-  da new --help
-
-  Usage: da new PROJECT_PATH
-    Create a new project
-
-  Available options:
-    -h,--help                Show this help text
-    PROJECT_PATH             Path to the new project. Name of the last folder will
-                             be the name of the new project.
-
-Developing with the SDK Assistant
-*********************************
-
-To begin, create a new DAML SDK project using the SDK Assistant. A project consists
-of a folder with a valid ``da.yaml`` file that, among other things, specifies
-which SDK release is being used. This is important because the release determines which
-versions of DAML and the DAML Sandbox the project code uses.
-
-In the SDK
-Assistant, create a DA project with this command::
+Global config file
+==================
 
-  da new <project_path>
+The ``daml`` configuration file, ``daml.yaml`` is in TODO directory.
```

Comment:
@associahedron, I'm gonna need your help filling in the information for a bunch of these. I'm gonna tag you in those places - could you either comment with the info or just push to the branch if it's easy to add?